src/components/Footer.js

```diff
@@ -133,6 +133,7 @@ const FooterWrapper = styled.footer`
       pointer-events: none;
       font-size: 0;
       opacity: 0;
+      display: none;
```

Comment:
Adding this property creates a bug on the hover state on the footer logo. While I was testing locally I also could still tab to that. Adding that was a last minute push, it would probably be better off being conditionally rendered in some way.